src/game/utils/score.c

```diff
@@ -334,6 +334,7 @@ int chr_score_clone(chr_score *src, chr_score *dst) {
         score_text t2;
         memcpy(&t2, t, sizeof(score_text));
         t2.text = create_text_obj(text_c(t->text));
+        text_set_horizontal_align(t2.text, text_get_horizontal_align(t->text));
```

Comment:
chr_score_t's `text *total;` is still experiencing alignment issues.

Reply:
possibly resolved.